security-info/security-goals.md

```diff
@@ -15,3 +15,5 @@ Qubes Security Goals
 Qubes implements a Security by Isolation approach by providing the ability to easily create many security domains. These domains are implemented as lightweight Virtual Machines (VMs) running under the Xen hypervisor. Qubes' main objective is to provide strong isolation between these domains, so that even if an attacker compromises one of the domains, the others are still safe. Qubes, however, does not attempt to provide any security isolation for applications running within the same domain. For example, a buggy web browser running in a Qubes domain could still be compromised just as easily as on a regular Linux distribution. The difference that Qubes makes is that now the attacker doesn't have access to all the software running in the other domains.
 
 Qubes also provides features that make it easy and convenient to run these multiple domains, such as seamless GUI integration into one common desktop, secure clipboard copy and paste between domains, secure file transfer between domains, disposable VMs, and much more. Qubes also provides an advanced networking infrastructure that allows for the creation of multiple network VMs which isolate all the world-facing networking stacks and proxy VMs which can be used for advanced VPN configurations and tunneling over untrusted connections.
+
+Qubes does not attempt to eliminate cooperative covert channels between two or more compromised domains. We don't believe this is possible to achieve on x86 hardware, and we also doubt it makes any sense in practice for most users -- after all, if the two domains are compromised, then it's probably already all lost anyway.
```

Comment:
This is definitely wrong. As @marmarek said (https://github.com/QubesOS/qubes-doc/pull/354#discussion_r113901427), just having two compromised domains tells you nothing about the whole Qubes system or the other domains.